sphinx/builders/html/__init__.py

```diff
@@ -240,10 +241,10 @@ def _get_translations_js(self) -> str:
         candidates = [path.join(dir, self.config.language,
                                 'LC_MESSAGES', 'sphinx.js')
                       for dir in self.config.locale_dirs] + \
-                     [path.join(package_dir, 'locale', self.config.language,
-                                'LC_MESSAGES', 'sphinx.js'),
-                      path.join(sys.prefix, 'share/sphinx/locale',
-                                self.config.language, 'sphinx.js')]
+            [path.join(package_dir, 'locale', self.config.language,
+                       'LC_MESSAGES', 'sphinx.js'),
+             path.join(sys.prefix, 'share/sphinx/locale',
+                       self.config.language, 'sphinx.js')]
```

Comment:
Why unindented?

Reply:
Probably auto formatter, I will revert this